### PR TITLE
Fix extension point

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Fix the condition to display the right data when user does not have enough credit.

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: extension_point_customercredit_checkout/react/index.js [hunk 1]
```diff
@@ -15,8 +15,12 @@ class CreditControlCheckout extends Component {
 
     this.state = {
       deadline: null,
+      mounted: false,
     }
   }
+  componentDidMount() {
+    this.setState({ mounted: true })
```
Comment (klzns) [inline extension_point_customercredit_checkout/react/index.js]: Nao precisa ser state, se for state ele vai re-renderizar o componente.

Pode simplesmente fazer:

```js
this.mounted = true
```

File: extension_point_customercredit_checkout/react/index.js [hunk 2]
```diff
@@ -38,6 +44,10 @@ class CreditControlCheckout extends Component {
     this.props.sendAuthorizationToFinish(true, [paymentFields])
   }
 
+  componentWillUnmount() {
+    this.setState({ mounted: false })
```
Comment (klzns) [inline extension_point_customercredit_checkout/react/index.js]:
```suggestion
    this.mounted = false
```

File: extension_point_customercredit_checkout/react/index.js [hunk 3]
```diff
@@ -26,9 +30,11 @@ class CreditControlCheckout extends Component {
   }
 
   handleInstallmentChange = e => {
+    const { mounted } = this.state
```
Comment (klzns) [inline extension_point_customercredit_checkout/react/index.js]:
```suggestion
```

File: extension_point_customercredit_checkout/react/index.js [hunk 4]
```diff
@@ -26,9 +30,11 @@ class CreditControlCheckout extends Component {
...
     const info = this.infoFromEvent(e)
     const chosenDeadline = parseInt(info.chosenDeadline)
-    this.setState({ deadline: parseInt(info.chosenDeadline) })
+
+    mounted && this.setState({ deadline: parseInt(info.chosenDeadline) })
```
Comment (klzns) [inline extension_point_customercredit_checkout/react/index.js]:
```suggestion
    this.mounted && this.setState({ deadline: parseInt(info.chosenDeadline) })
```

File: extension_point_customercredit_checkout/react/index.js [hunk 5]
```diff
@@ -15,8 +15,12 @@ class CreditControlCheckout extends Component {
 
     this.state = {
       deadline: null,
+      mounted: false,
```
Comment (klzns) [inline extension_point_customercredit_checkout/react/index.js]:
```suggestion
```

Comment (kevinch) [reply extension_point_customercredit_checkout/react/index.js]: Good catch.

**Pull request merged**